xml/admin_caasp_cephtoolbox.xml

```diff
@@ -190,8 +190,8 @@
       spec:
         initContainers:
         - name: config-init
-          image: rook/ceph:v1.4.4
-          command: ["/usr/local/bin/toolbox.sh"]
+          image: registry.suse.com/ses/7/rook/ceph:1.4.6
```

Comment:
```suggestion
          image: registry.suse.com/ses/7/rook/ceph:<replaceable>LATEST_TAG</replaceable>
```

Reply:
Yes, that looks better. Thanks for the suggestion.